userpanel/modules/notices/style/bclean/templates/notices.html

```diff
@@ -26,11 +26,10 @@
 					<td>{$items.subject}</td>
 					<td>
 						{if $items.type == $smarty.const.MSG_USERPANEL_URGENT || $items.type == $smarty.const.MSG_USERPANEL}
-							{if $items.status == $smarty.const.MSG_SENT && !$items.lastreaddate}
-								{trans("Unread")}
-							{elseif $items.status == $smarty.const.MSG_SENT || $items.status == $smarty.const.MSG_DELIVERED}
-								{trans("Was read on:")}<br />
+							{if $items.status == $smarty.const.MSG_SENT || $items.status == $smarty.const.MSG_DELIVERED}
 								{$items.lastreaddate|date_format:"%Y/%m/%d %H:%M"}
+							{else}
+								{trans("Unread")}
 							{/if}
```

Comment:
Tu widzę zmianę logiki - nie wiem czy więcej warunków w pierwotnym kodzie nie było celowe.

Reply:
No dobrze to puszczę to oddzielnym PR a tą zmianę cofam na tą chwilę.